apps/remote_control/lib/lexical/remote_control/search/indexer/source/reducer.ex

```diff
@@ -13,7 +13,7 @@ defmodule Lexical.RemoteControl.Search.Indexer.Source.Reducer do
   alias Lexical.RemoteControl.Search.Indexer.Metadata
   alias Lexical.RemoteControl.Search.Indexer.Source.Block
 
-  defstruct [:analysis, :entries, :position, :ends_at, :blocks, :block_hierarchy]
+  defstruct [:analysis, :entries, :position, :blocks, :block_hierarchy, extractors: []]
```

Comment:
I noticed that the default value for `extractors` in many functions is nil, but here it is `[]`. Is there any particular reason for this